<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppInfoPopover</class>
 <widget class="QWidget" name="AppInfoPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>App</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLabel" name="summaryLabel">
         <property name="text">
          <string notr="true">TextLabel</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="iconLabel">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="appNameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QWidget" name="actionsWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="tCircularSpinner" name="actionSpinner" native="true"/>
          </item>
          <item>
           <widget class="QPushButton" name="installButton">
            <property name="text">
             <string>Install</string>
            </property>
            <property name="icon">
             <iconset theme="package-install"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="launchButton">
            <property name="text">
             <string>Launch</string>
            </property>
            <property name="icon">
             <iconset theme="go-next"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>Remove</string>
            </property>
            <property name="icon">
             <iconset theme="package-remove"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="tConditionalWidget" name="progressWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="tStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="detailsPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>475</width>
            <height>202</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="ScreenshotBrowser" name="screenshotBrowser" native="true"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="rightMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="descriptionLabel">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="homepageButton">
                 <property name="text">
                  <string>Visit Website</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="donateButton">
                 <property name="text">
                  <string>Donate</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
   <slots>
    <signal>switchingFrame(int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tCircularSpinner</class>
   <extends>QWidget</extends>
   <header location="global">tcircularspinner.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tConditionalWidget</class>
   <extends>QWidget</extends>
   <header location="global">tconditionalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenshotBrowser</class>
   <extends>QWidget</extends>
   <header>popovers/screenshotbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
